packages/shared/src/db.ts

```diff
@@ -11,17 +11,38 @@ import {
   PostgresQueryCompiler,
 } from "kysely";
 import { DB } from ".";
+import { logger } from "./server";
 
 // Instantiated according to the Prisma documentation
 // https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices
 
 const prismaClientSingleton = () => {
-  return new PrismaClient({
-    log:
-      env.NODE_ENV === "development"
-        ? ["query", "error", "warn"]
-        : ["error", "warn"],
+  const client = new PrismaClient<
```

Comment:
We could also use a child logger here. That would allow us to add an attribute "prisma" to the structured output instead of using a prefix, which would simplify filtering for prisma logs within our monitoring tools.